<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EPOS2Widget</class>
 <widget class="QWidget" name="EPOS2Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EPOS2Widget</string>
  </property>
  <widget class="QGroupBox" name="manualControlGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Manual Control</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>135</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="nodeIDcomboBox">
       <item>
        <property name="text">
         <string>1 (Trans)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 (Pitch)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 (Yaw)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 (Roll)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nodeIDlabel">
       <property name="text">
        <string>Node ID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>nodeIDcomboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="moveRelButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Move RELATIVE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="moveAbsButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Move ABSOLUTE</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="haltButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Halt</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="targetQClabel">
       <property name="text">
        <string>Target QC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>targetQCspinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="homingButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Homing</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="enableNodeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="disableNodeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disable</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="targetQCspinBox">
       <property name="minimum">
        <number>-9999999</number>
       </property>
       <property name="maximum">
        <number>9999999</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="homeAllButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home All Axes</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="connectionGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>EPOS Connect/Disconnect</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnectButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="motorReadingsGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Motor Readings</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="pitchQClabel">
       <property name="text">
        <string>Pitch QC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="yawQClabel">
       <property name="text">
        <string>Yaw QC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="rollQClabel">
       <property name="text">
        <string>Roll QC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="transQClabel">
       <property name="text">
        <string>Translation QC</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="transStatusLabel">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="pitchStatusLabel">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="yawStatusLabel">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="rollStatusLabel">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="transQC_LCD">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="pitchQC_LCD">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="yawQC_LCD">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="rollQC_LCD">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>8</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="outputTextEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>16</y>
     <width>261</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="servoLoopGroupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>150</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Servo Loop</string>
   </property>
   <widget class="QPushButton" name="disableServoLoopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable Servo Loop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableServoLoopButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Servo Loop</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="trajectoryGroupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Trajectory</string>
   </property>
   <widget class="QPushButton" name="trajOpenFileButton">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>25</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="trajDriveButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>25</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Drive</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="trajStepLineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>25</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>0 of 0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>connectButton</tabstop>
  <tabstop>disconnectButton</tabstop>
  <tabstop>nodeIDcomboBox</tabstop>
  <tabstop>enableNodeButton</tabstop>
  <tabstop>disableNodeButton</tabstop>
  <tabstop>targetQCspinBox</tabstop>
  <tabstop>homingButton</tabstop>
  <tabstop>moveAbsButton</tabstop>
  <tabstop>haltButton</tabstop>
  <tabstop>moveRelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
